src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1489,6 +1454,79 @@ public static function debug_data() {
 		return $info;
 	}
 
+	/**
+	 * Populate the file system section of the debug data.
+	 *
+	 * Using the {@see 'debug_information'} filter for this.
+	 *
+	 * @since tbd
+	 *
+	 * @param array $info The debug information to be added to the core information page.
+	 *
+	 * @return array
+	 */
+	public static function wp_filesystem( array $info ): array {
```

Comment:
```suggestion
	private static function wp_filesystem( array $info ): array {
```
At this stage, the new methods should begin life as `private`, given that once they're `protected` or `public`, they must remain as such for backward compatibility purposes.